apps/settings/serializers/auth/ldap.py

```diff
@@ -103,10 +104,11 @@ class LDAPSettingSerializer(serializers.Serializer):
     AUTH_LDAP = serializers.BooleanField(required=False, label=_('LDAP'))
     AUTH_LDAP_SYNC_ORG_IDS = OrgListField()
 
-    def post_save(self):
-        keys = ['AUTH_LDAP_SYNC_IS_PERIODIC', 'AUTH_LDAP_SYNC_INTERVAL', 'AUTH_LDAP_SYNC_CRONTAB']
-        kwargs = {k: self.validated_data[k] for k in keys if k in self.validated_data}
-        if not kwargs:
-            return
+    periodic_key = 'AUTH_LDAP_SYNC_IS_PERIODIC'
+    interval_key = 'AUTH_LDAP_SYNC_INTERVAL'
+    crontab_key = 'AUTH_LDAP_SYNC_CRONTAB'
+
+    @staticmethod
+    def import_task_function(**kwargs):
         from settings.tasks import import_ldap_user_periodic
         import_ldap_user_periodic(**kwargs)
```

Comment:
I need more information to provide an accurate answer. Are you asking about any particular part of this code or just general code differences between versions? Please be specific with what aspects of the code you want to compare or which version(s) you would like me to analyze.